### DISCUSSION TITLE
Accept prop

### DISCUSSION THREAD

**Discussion by KamilSlaweta** [react-dropzone/react-dropzone#1315] [Q&A]
Is that expected that passing 

```
const defaultAccept = {
  'image/*': ['.jpeg', '.jpg', '.png'],
};

```

would result in not being able to upload, for instance, `.heic` img but there are not problems to upload all `.avif` or `tiff` images?

**Comment by rolandjitsu** [comment]
@KamilSlaweta the accept will filter out anything that's not an image with those extensions. That is also passed down to the input/https://developer.mozilla.org/en-US/docs/Web/API/Window/showOpenFilePicker, so it depends on the browser as well.
